lib/segment/src/data_types/vectors.rs

```diff
@@ -12,6 +12,111 @@ use crate::vector_storage::query::context_query::ContextQuery;
 use crate::vector_storage::query::discovery_query::DiscoveryQuery;
 use crate::vector_storage::query::reco_query::RecoQuery;
 
+pub mod internal {
```

Comment:
Definition of segment internal vector is here

Reply:
We don't really use this pattern anywhere, except for tests. But at the same time, I don't have a strong objection against it.

Reply:
it's temporary. In next PR's about this issue I gonna move REST vector to collection crate